site2/website-next/contribute/document-preview.md

```diff
@@ -37,14 +37,22 @@ Follow these steps to preview documentation changes on the **master** branch.
 2. Run the following command to preview changes:
 
     ```bash
-    # Preview the next version
+    # Preview changes on latest stable version 
     sh start.sh
-    # Preview historical versions
-    sh start.sh [<version-number> ...]
+
+    # Preview changes on a specific version version
+    sh start.sh 2.10.x 
+
+    # Preview changes on multiple versions
+    sh start.sh 2.10.x 2.9.x ...
     ```
 
 3. By default, a browser window will open at http://localhost:3000 to show the changes.
```

Comment:
Suggest removing the numbering `3` here. It's the result of step 2, not user's behavior.